crates/bevy_render/src/rangefinder.rs

```diff
@@ -0,0 +1,52 @@
+use bevy_math::{Mat4, Vec4};
+
+use crate::view::ExtractedView;
+
+pub struct ViewRangefinder3d {
+    inverse_view_row_2: Vec4,
+}
+
+/// A helper for calculating the draw order of meshes.
+impl ViewRangefinder3d {
+    pub fn from_view(view: &ExtractedView) -> ViewRangefinder3d {
+        let inverse_view_matrix = view.transform.compute_matrix().inverse();
+        ViewRangefinder3d {
+            inverse_view_row_2: inverse_view_matrix.row(2),
+        }
+    }
+
+    /// Calculates the view-space Z value for a mesh's origin
+    pub fn distance(&self, mesh_transform: &Mat4) -> f32 {
```

Comment:
I think this is too specific; there's no reason the transform must belong to a mesh.

Reply:
Removed mention of meshes from the parameter name and doc comment.